5 Object Oriented Programming/10 fourPillarsOfOOP.js

```diff
@@ -1,10 +1,110 @@
-/** FOUR PILLARS OF OBJECT ORIENTED PROGRAMMING
- * - ENCAPSULATION
- * - ABSTRACTION
- * - INHERITANCE
- * - POLYMORPHISM
- * 
- * 
- * To add code for each feature - open for contributions
- * Will accept Code that explains all the four pillars with code and proper comments.
-*/
\ No newline at end of file
+/* RESOURCES
+   https://www.javatpoint.com/javascript-oops-abstraction
+   https://www.javatpoint.com/javascript-oops-polymorphism
+   https://www.javatpoint.com/javascript-oops-inheritance
+   https://www.javatpoint.com/javascript-oops-encapsulation
+   https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/Object-oriented_JS */
+   
+
+// FOUR PILLARS OF OBJECT ORIENTED PROGRAMMING
```

Comment:
Oops... Please have a look at [SampleJS.md](https://github.com/MadaZZ/JS-Concepts/blob/master/SampleJS.md)

Reply:
Done! Is it fine now?